substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1880,6 +1881,28 @@ impl<T: Config> StakingInterface for Pallet<T> {
 		}
 	}
 
+	fn force_withdraw(stash: &Self::AccountId, amount: Self::Balance) -> DispatchResult {
+		let mut ledger = Self::ledger(Stash(stash.clone()))?;
+		ledger.active = ledger.active.saturating_sub(amount);
+		// FIXME eagr should we check if active >= ED before updating?
```

Comment:
You don't strictly need to since `saturating_sub` would just make it 0. But probably a good defensive measure to add is to ensure `force_withdraw` can only be done for `0 < amount < ED`.

Reply:
Oh, I mean do we need to

```rs
ledger.active = ledger.active.saturating_sub(amount);
// check before updating the ledger
ensure!(ledger.active >= T::Currency::minimum_balance());
ledger.update()?;
```